pvlib/irradiance.py

```diff
@@ -1516,24 +1528,24 @@ def ghi_from_poa_driesse_2023(surface_tilt, surface_azimuth,
     solar_azimuth : numeric
         Solar azimuth angle. [degree]
     poa_global : numeric
-        Plane-of-array global irradiance, aka global tilted irradiance. [W/m^2]
+        Plane-of-array global irradiance, aka global tilted irradiance. [Wm⁻²]
     dni_extra : numeric, optional
-        Extraterrestrial direct normal irradiance. [W/m^2]
+        Extraterrestrial direct normal irradiance. [Wm⁻²]
```

Comment:
How do these changes relate to #2209?  Should we stick with one approach over the other?

Reply:
I got the impression from the discussion in the related issue (#2205) that Wm⁻² was the prefered formatting but the problem was how to type that out, so I figured if #2209 gets merged, since that addresses the inconvenience of typing it out, then we would slowly move towards Wm⁻². That's the change here, although in this case I did just type it out.
I am not opposed to reverting the change for now if that keeps this PR more focused though (following up on the useful advice in one of your other comments)